### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by Phips0812**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -2,11 +2,20 @@
 
 *****************
 
-## Release 1.0.0 - First puplic release
+## Release ONDEWO Proto Compiler 1.1.0 
+
+### New Features
+* Proto compiler for Angular now uses `ngx-grpc` 2.1.0 instead of 0.x.x
```
Comment (gaborod16) [inline RELEASE.md:8]: :1st_place_medal:

File: angular/example/package.json [hunk 2]
```diff
@@ -1,22 +1,17 @@
 {
   "name": "@ondewo/angular-proto-compiler-example",
-  "version": "0.1.0",
-  "description": "Example for how to use the ondewo proto angular compiler",
+  "version": "2.0.0",
```
Comment (gaborod16) [inline angular/example/package.json:3]: shouldn't it be `1.1.0`?

**Review by gaborod16 — APPROVED** [review]
Small version mismatch.
Looks great!

File: angular/example/package.json (hunk 2 above)
Comment (Phips0812) [reply angular/example/package.json:3]: this is just an example so it doesn't matter in the end

**Pull request merged**